sr_hand_health_report/src/sr_hand_health_report/sr_hand_health_report_check.py

```diff
@@ -123,15 +168,22 @@ def __init__(self, hand_side, fingers_to_test):
                                      "wrj1": -1, "wrj2": 1}
 
     def _init_map_finger_joints(self):
+        """
+            Initializes the dictionary for joints to fingers.
+        """
         fingers_to_joint_map = OrderedDict()
```

Comment:
By default in python3, all dicts are ordered. Or are you using a specific feature of this object that is not available in classical dicts?

Reply:
I can replace it if you wish, but this OrderedDict part was not added by me. Usually, I am trying not to mess with someone's implementation if not necessary.

I haven't seen anywhere specific features of this being used, so should be fine?

Reply:
Fine function-wise yes, but it avoids unnecessary imports and less confusion for maintainability :) Not sure if it's part of the scope of the story.... @toliver  any idea?

Reply:
I replaced with {}, I couldn't spot any malfunctioning tests so lets remove OrderedDict